.github/ISSUE_TEMPLATE/libmamba-bug-report.yml

```
@@ -0,0 +1,84 @@
+name: Libmamba Bug Report (Experimental Feature)
+description: Create a report for the experimental solver
+labels: ['solver::libmamba', 'tag::experimental']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for helping conda! 🐍
+
+        You are trying out an experimental feature. If you are encountering blocking issues we first suggest removing the experimental solver by typing the following code into your terminal:
+        ```bash
+        $ conda remove conda-libmamba-solver
+        ```
+
+        Going through bugs and issues takes up a lot of time, so please be so kind and take a few minutes to fill out all the areas to the best of your ability.
+
+        There will always be more issues than there is time to do them, and so we will need to selectively close issues that don't provide enough information, so we can focus our time on helping people like you who fill out the issue form completely. Thank you for your collaboration!
+
+        There are also already a lot of open issues, so please take 2 minutes and search through existing ones to see if what you are experiencing already exists
+
+        Thanks for helping conda be amazing. We are nothing without people like you helping build a better community 💐!
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other issues and couldn't find a solution or duplication
+          required: true
+        - label: I looked at the [README.md](https://github.com/conda-incubator/conda-libmamba-solver) and didn't see anything to help
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: And what should have happened instead? This really helps us triage quicker and greatly increases the chance that we can get around to solve your issue
+      placeholder: Tell us what you see!
+      value: "A wild Caterpie appeared!"
```

```suggestion
      value: "I tried to install package XYZ and these things happened instead:"
```
I'm not sure how to say this, but could we exchange this with something like this to reduce the risk of confusion?

---

I'm also not sure we need any placeholder text at all here? Just make sure it's a required field so they can't leave it empty?

---

I'm... not sure